src/Extension/MemberExtension.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SilverStripe\SecurityExtensions\Extension;
+
+use SilverStripe\Forms\CheckboxField;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\ORM\FieldType\DBDatetime;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
+
+/**
+ * Extend Member to add relationship to registered methods and track some specific preferences
+ *
+ * @property Member|MemberExtension owner
+ */
+class MemberExtension extends DataExtension
+{
+    public function updateCMSFields(FieldList $fields)
+    {
+        $currentUser = Security::getCurrentUser();
+
+        // We can allow an admin to require a user to change their password however. But:
+        // - Don't show a read only field if the user cannot edit this record
+        // - Don't show if a user views their own profile (just let them reset their own password)
+        if ($currentUser && ($currentUser->ID !== $this->owner->ID) && $this->owner->canEdit()) {
+            $requireNewPassword = CheckboxField::create(
+                'RequirePasswordChangeOnNextLogin',
+                _t(__CLASS__ . 'RequirePasswordChangeOnNextLogin', 'Require password change on next login')
+            );
+            $fields->insertAfter('Password', $requireNewPassword);
+
+            $fields->dataFieldByName('Password')->addExtraClass('form-group--no-divider');
```

Comment:
Do you think it might be better to add a unique class to this field and style that instead?

Reply:
This is a unique class, to my knowledge.

Reply:
Continue discussion in https://github.com/silverstripe/silverstripe-security-extensions/pull/2/files#r291015985